src/ripple/app/misc/impl/ValidatorSite.cpp

```diff
@@ -294,6 +294,9 @@ ValidatorSite::onSiteFetch(
     }
     else
```

Comment:
I just realized we probably also want to set `lastRefreshStatus` as `invalid` in the `if` for this `else`
https://github.com/ripple/rippled/blob/develop/src/ripple/app/misc/impl/ValidatorSite.cpp#L208-L214